### ISSUE TITLE
Submitting, author notification and admin notification don't work. Ps 1.7.3.4 - 1.7.4.1

### ISSUE THREAD

**Issue by Apury** [PrestaShop/contactform#26]
Hello there. 

I'm running PS 1.7.3.4 with contactform 4.1.1 on the default theme. 

If I try to submit a message, I get the default error message. This is fixable by adding 

```
<input type="hidden" name="url" value=""/>
<input type="hidden" name="token" value="{$token}" />
```
before the submit button. 

I have php 7.1 IMAP installed and properly configured. However, once I activate the feature of sending an e-mail to my admin account when somebody submits a form or the feature of sending the confirmation back to the author, the form gives back the default error message.

It doesn't matter whether I activate either of the two features or both at once. 

p.s. I've installed a fresh installation of p.s. 1.7.4.1 without anything else. If I submit a contact form, I get greeted by: 

> An error occurred while sending the message.
> An error occurred while sending the message.

**Comment by matks** [comment]
Hello and thanks for the bug report !

So to make sure I understand correctly:
- there is a bug for PS 1.7.3.4 + contactform 4.1.1
- there is a 2nd bug for PS 1.7.4.1 + contactform 4.1.1

Is that right ?

**Comment by Apury** [comment]
There isn't necessarily a 2nd bug for PS 1.7.4.1. I've just chosen 1.7.4.1
to reproduce it. Neither version allows me to receive contact form
notifications, nor sends the initial confirmation / new replies to the
customer's e-mail.

I've created http://forge.prestashop.com/browse/BOOM-6004 and will supply
it with a whole bunch of images in the next minutes.

**Comment by Apury** [comment]
I've fixed it for my 1.7.3.4 case. 

The problem was the 4.1.1 and prioir contactform.php. I've used the 3.0 version, added the confirm customer owns order and GDRP hook manually and now the form works flawlessly and sends confirmation e-mails for having received the form aswell as anotification to the admin. 

I pressume, to send the reply that was written via the customer service part of the BO a IMAP account is required? If not, then that's the only part of the module that isn't working.
